buildSrc/build.gradle

```diff
@@ -10,6 +10,12 @@ repositories {
     jcenter()
 }
 
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+sourceSets.main.java.srcDirs = []
```

Comment:
Currently we need joint compilation of both groovy and java so they need to be in the same sourceset. As more classes move over to java, we will remove the groovy sourceset entirely